### ISSUE TITLE
svrx 命令失败

### ISSUE THREAD

**Issue by mq931001** [svrxjs/svrx#137]
C:\Users\DELL\AppData\Roaming\npm\node_modules\@svrx\cli\node_modules\@svrx\util\lib\rc-file-read.js:38
  return { ...globalConfig, ...scopeConfig };
           ^^^

SyntaxError: Unexpected token ...
    at createScript (vm.js:74:10)
    at Object.runInThisContext (vm.js:116:10)
    at Module._compile (module.js:533:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\DELL\AppData\Roaming\npm\node_modules\@svrx\cli\node_modules\@svrx\util\lib\svrx-util.js:1:84)

node 版本8.2.1

**Comment by int64ago** [comment]
TL;DR

Node >= 8.9.0

FYI https://github.com/svrxjs/svrx/pull/63
